PR problem: docstring error

Hi!

I'm having trouble getting the CI to pass on this PR: https://github.com/theislab/scanpy/pull/1476

This seems to be the issue:

```
>           raise SyntaxError(msg, (filename, lineno, 2, text))
E             File "/home/travis/build/theislab/scanpy/scanpy/external/pp/_scrublet.py", line 9
E               >   <
E                ^
E           SyntaxError: Header of function `scanpy.external.pp._scrublet.scrub_doublets`’s docstring should start with one-line description:
E           
E           ␣␣␣␣"""\
E           ␣␣␣␣My one-line␣description.
E           
E           ␣␣␣␣…
E           ␣␣␣␣"""
scanpy/tests/test_docs.py:38: SyntaxError
```

... but the function does have that one-line description. 

Would you be able to suggest to me what I'm doing wrong?

--- a/scanpy/tests/test_docs.py
+++ b/scanpy/tests/test_docs.py
@@ -24,13 +24,16 @@ def test_function_headers(f):
     broken = [i for i, l in enumerate(lines) if l.strip() and not l.startswith("    ")]
     if any(broken):
         msg = f'''\
-Header of function `{name}`’s docstring should start with one-line description:
+Header of function `{name}`’s docstring should start with one-line description
+and be consistently indented like this:
 
 ␣␣␣␣"""\\
 ␣␣␣␣My one-line␣description.
 
 ␣␣␣␣…
 ␣␣␣␣"""
+
+The displayed line is under-indented.
 '''
         filename = inspect.getsourcefile(f)
         _, lineno = inspect.getsourcelines(f)

--- a/scanpy/tests/test_docs.py
+++ b/scanpy/tests/test_docs.py
@@ -21,7 +21,7 @@ def test_function_headers(f):
     assert f.__doc__ is not None, f"{name} has no docstring"
     lines = getattr(f, "__orig_doc__", f.__doc__).split("\n")
     assert lines[0], f"{name} needs a single-line summary"
-    broken = [i for i, l in enumerate(lines) if l and not l.startswith("    ")]
+    broken = [i for i, l in enumerate(lines) if l.strip() and not l.startswith("    ")]
     if any(broken):
         msg = f'''\
 Header of function `{name}`’s docstring should start with one-line description:

